main/templates/main/landing.html

```diff
@@ -1,13 +1,12 @@
 {% extends 'main/layout.html' %}
 
-{% block title %}Mataroa — Blogging platform for minimalists{% endblock %}
+{% block title %}{{ instance_name }} - {{ instance_description }}{% endblock %}
 
 {% block content %}
 <main>
-    <h1>Mataroa blog</h1>
-    <p>
-        Naked blogging platform, for minimalists. <em>Just write.</em>
-    </p>
+    <h1>{{ instance_name }}</h1>
+    <p>{{ instance_description }} <em>Just write.</em></p>
```

Comment:
this is kinda annoying. having the `<em>Just write.</em>` as well :| not sure how to improve though